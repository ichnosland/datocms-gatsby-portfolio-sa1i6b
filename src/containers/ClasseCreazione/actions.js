/*
 *
 * ClasseCreazione actions
 *
 */

import {
  CLASSE_CREAZIONE_SCUOLE_ATTIVE_FETCH,
  CLASSE_CREAZIONE_SCUOLE_ATTIVE_SET,
  CLASSE_CREAZIONE_GEO_PROVINCIA_FETCH,
  CLASSE_CREAZIONE_GEO_SET,
  CLASSE_CREAZIONE_GEO_COMUNI_FETCH,
  CLASSE_CREAZIONE_GEO_COMUNI_SET,
  CLASSE_CREAZIONE_DATA_SET,
  CLASSE_CREAZIONE_DATA_POST,
  CLASSE_CREAZIONE_RESET,
  CLASSE_CREAZIONE_SPINNER_SET,
  CLASSE_CREAZIONE_FEEDBACK_SET,
  CLASSE_CREAZIONE_FEEDBACK_RESET,
  CLASSE_CREAZIONE_GEO_SCUOLE_SET,
} from './constants';


export const classeCreazioneScuoleAttiveFetch = () => ({
  type: CLASSE_CREAZIONE_SCUOLE_ATTIVE_FETCH,
});

export const classeCreazioneScuoleAttiveSet = (payload) => ({
  type: CLASSE_CREAZIONE_SCUOLE_ATTIVE_SET,
  payload,
});

export const classeCreazioneGeoProvinciaFetch = () => ({
  type: CLASSE_CREAZIONE_GEO_PROVINCIA_FETCH,
});

export const classeCreazioneGeoSet = (payload) => ({
  type: CLASSE_CREAZIONE_GEO_SET,
  payload,
});

export const classeCreazioneGeoComuneFetch = (sigla) => ({
  type: CLASSE_CREAZIONE_GEO_COMUNI_FETCH,
  sigla,
});

export const classeCreazioneGeoComuneSet = (payload) => ({
  type: CLASSE_CREAZIONE_GEO_COMUNI_SET,
  payload,
});

export const classeCreazioneGeoScuoleSet = (payload) => ({
  type: CLASSE_CREAZIONE_GEO_SCUOLE_SET,
  payload,
});

export const classeCreazioneDataSet = (payload) => ({
  type: CLASSE_CREAZIONE_DATA_SET,
  payload,
});

export const classeCreazioneDataPost = (payload, history) => ({
  type: CLASSE_CREAZIONE_DATA_POST,
  payload,
  history,
});

export const classeCreazioneReset = () => ({
  type: CLASSE_CREAZIONE_RESET,
});

export const classeCreazioneSpinnerSet = (enable) => ({
  type: CLASSE_CREAZIONE_SPINNER_SET,
  enable,
});

export const classeCreazioneFeedbackSet = (enable, tipologia, messaggio) => ({
  type: CLASSE_CREAZIONE_FEEDBACK_SET,
  enable,
  tipologia,
  messaggio,
});

export const classeCreazioneFeedbackReset = () => ({
  type: CLASSE_CREAZIONE_FEEDBACK_RESET,
});
