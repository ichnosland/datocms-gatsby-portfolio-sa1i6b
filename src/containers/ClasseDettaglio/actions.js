/*
 *
 * ClasseDettaglio actions
 *
 */

import {
  CLASSE_DETTAGLIO_ESPELLI,
  CLASSE_DETTAGLIO_RESET,
  CLASSE_DETTAGLIO_DATA_FETCH,
  CLASSE_DETTAGLIO_CORSO_FETCH,
  CLASSE_DETTAGLIO_CONTENUTO_SET,
  CLASSE_DETTAGLIO_DISPLAY_SET,
  CLASSE_DETTAGLIO_DISPLAY_SORT,
  CLASSE_DETTAGLIO_FEEDBACK_SET,
  CLASSE_DETTAGLIO_FEEDBACK_RESET,
  CLASSE_DETTAGLIO_SPINNER_SET,
} from './constants';

export const classeDettaglioEspelli = (payload) => ({
  type: CLASSE_DETTAGLIO_ESPELLI,
  payload,
});

export const classeDettaglioReset = () => ({
  type: CLASSE_DETTAGLIO_RESET,
});

export const classeDettaglioCorsoFetch = (payload) => ({
  type: CLASSE_DETTAGLIO_CORSO_FETCH,
  payload,
});

export const classeDettaglioDataFetch = (payload) => ({
  type: CLASSE_DETTAGLIO_DATA_FETCH,
  payload,
});

export const classeDettaglioContenutoSet = (payload) => ({
  type: CLASSE_DETTAGLIO_CONTENUTO_SET,
  payload,
});

export const classeDettaglioDisplaySort = (payloadData, sortingData) => ({
  type: CLASSE_DETTAGLIO_DISPLAY_SORT,
  payloadData,
  sortingData,
});

export const classeDettaglioDisplaySet = (payload) => ({
  type: CLASSE_DETTAGLIO_DISPLAY_SET,
  payload,
});

export const classeDettaglioFeedbackSet = (hasFeedback, tipologia, messaggio) => ({
  type: CLASSE_DETTAGLIO_FEEDBACK_SET,
  hasFeedback,
  tipologia,
  messaggio,
});

export const classeDettaglioFeedbackReset = () => ({
  type: CLASSE_DETTAGLIO_FEEDBACK_RESET,
});

export const classeDettaglioSpinnerSet = (enable) => ({
  type: CLASSE_DETTAGLIO_SPINNER_SET,
  enable,
});
