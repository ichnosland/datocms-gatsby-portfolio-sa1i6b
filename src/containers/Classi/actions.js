/*
 *
 * Classi actions
 *
 */

import {
  CLASSI_DATA_FETCH,
  CLASSI_DATA_SET,
  CLASSI_DATA_RESET,
  CLASSI_FEEDBACK_RESET,
  CLASSI_FEEDBACK_SET,
  CLASSI_SPINNER_SET,
} from './constants';

export const classiDataFetch = (idDisciplina) => ({
  type: CLASSI_DATA_FETCH,
  idDisciplina,
});

export const classiDataSet = (payload) => ({
  type: CLASSI_DATA_SET,
  payload,
});

export const classiDataReset = () => ({
  type: CLASSI_DATA_RESET,
});

export const classiFeedbackReset = () => ({
  type: CLASSI_FEEDBACK_RESET,
});

export const classiFeedbackSet = (enable = true, message = '', errorData = {}) => ({
  type: CLASSI_FEEDBACK_SET,
  enable,
  message,
  errorData,
});

export const classiSpinnerSet = (enable) => ({
  type: CLASSI_SPINNER_SET,
  enable,
});
