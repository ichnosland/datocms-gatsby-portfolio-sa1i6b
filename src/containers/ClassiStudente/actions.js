/*
 *
 * ClassiStudente actions
 *
 */

import {
  CLASSI_STUDENTE_FETCH,
  CLASSI_STUDENTE_SET,
  CLASSI_STUDENTE_RESET,
  CLASSI_STUDENTE_ISCRIVITI_POST,
  CLASSI_STUDENTE_SPINNER_SET,
  CLASSI_STUDENTE_FEEDBACK_SET,
  CLASSI_STUDENTE_FEEDBACK_RESET,
} from './constants';

export const classiStudenteFetch = (disciplinaId) => ({
  type: CLASSI_STUDENTE_FETCH,
  disciplinaId,
});

export const classiStudenteSet = (payload) => ({
  type: CLASSI_STUDENTE_SET,
  payload,
});

export const classiStudenteReset = () => ({
  type: CLASSI_STUDENTE_RESET,
});

export const classiStudenteIscrivitiPost = (codiceCorso, configuration) => ({
  type: CLASSI_STUDENTE_ISCRIVITI_POST,
  codiceCorso,
  configuration,
});

export const classiStudenteSpinnerSet = (enable) => ({
  type: CLASSI_STUDENTE_SPINNER_SET,
  enable,
});

export const classiStudenteFeedbackSet = (enable, tipologia, messaggio) => ({
  type: CLASSI_STUDENTE_FEEDBACK_SET,
  enable,
  tipologia,
  messaggio,
});

export const classiStudenteFeedbackReset = () => ({
  type: CLASSI_STUDENTE_FEEDBACK_RESET,
});
