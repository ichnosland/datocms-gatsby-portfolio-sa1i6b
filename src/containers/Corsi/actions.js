/*
 *
 * Corsi actions
 *
 */

import {
  CORSI_RESET,
  CORSI_ERROR_SET,
  CORSI_ERROR_RESET,
  CORSI_SPINNER_SET,
  CORSI_LIST_INITIALIZE,
  CORSI_LIST_DOCENTE_FETCH,
  CORSI_LIST_STUDENTE_FETCH,
  CORSI_LIST_SET,
  CORSI_CORSO_DETAIL_DOCENTE_TRIGGER,
  CORSI_CORSO_DETAIL_DOCENTE_SET,
} from './constants';

export const corsiReset = () => ({
  type: CORSI_RESET,
});

export const corsiSpinnerSet = (enable) => ({
  type: CORSI_SPINNER_SET,
  enable,
});

export const corsiErrorSet = (enable = true, message = '', errorData = {}) => ({
  type: CORSI_ERROR_SET,
  enable,
  message,
  errorData,
});

export const corsiErrorReset = () => ({
  type: CORSI_ERROR_RESET,
});

export const corsiListInitialize = (configuration, isDocente, userId) => ({
  type: CORSI_LIST_INITIALIZE,
  configuration,
  isDocente,
  userId,
});

export const corsiListDocenteFetch = (idDisciplina) => ({
  type: CORSI_LIST_DOCENTE_FETCH,
  idDisciplina,
});

export const corsiListStudenteFetch = (idDisciplina) => ({
  type: CORSI_LIST_STUDENTE_FETCH,
  idDisciplina,
});

export const corsiListSet = (payload) => ({
  type: CORSI_LIST_SET,
  payload,
});

export const corsiCorsoSelectedDocenteTrigger = (idCorso, configuration, isDocente, userId) => ({
  type: CORSI_CORSO_DETAIL_DOCENTE_TRIGGER,
  idCorso,
  configuration,
  isDocente,
  userId,
});

export const corsiCorsoDetailDocenteSet = (payload) => ({
  type: CORSI_CORSO_DETAIL_DOCENTE_SET,
  payload,
});
