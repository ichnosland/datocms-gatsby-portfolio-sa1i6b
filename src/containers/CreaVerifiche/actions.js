/*
 *
 * CreaVerifiche actions
 *
 */

import {
  CREA_VERIFICHE_FETCH,
  CREA_VERIFICHE_FETCH_ERROR,
  CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH,
  CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH_ERROR,
  CREA_VERIFICHE_SET,
  CREA_VERIFICHE_SPINNER_SET,
  CREA_VERIFICHE_POST,
  CREA_VERIFICHE_POST_ERROR,
  CREA_VERIFICHE_VERIFICA_SET,
  CREA_VERIFICHE_VERIFICA_ESERCIZIO_ADD,
  CREA_VERIFICHE_VERIFICA_ESERCIZIO_REMOVE,
  CREA_VERIFICHE_VERIFICA_RIEPILOGO_VIEW,
} from './constants';

export const creaverificheVerificaPostError = (messaggio = 'Impossibile inviare i dati') => ({
  type: CREA_VERIFICHE_POST_ERROR,
  messaggio,
});

export const creaverificheVerificaPost = (payload) => ({
  type: CREA_VERIFICHE_POST,
  payload,
});

export const creaverificheVerificaEserciziFetch = (prerequisito) => ({
  type: CREA_VERIFICHE_FETCH,
  prerequisito,
});

export const creaverificheVerificaEserciziFetchError = (messaggio = 'Impossibile caricare l\'elenco degli esercizi disponibili') => ({
  type: CREA_VERIFICHE_FETCH_ERROR,
  messaggio,
});

export const creaverificheVerificaEsercizioPreviewFetch = (esercizioId) => ({
  type: CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH,
  esercizioId,
});

export const creaverificheVerificaEsercizioPreviewFetchError = (
  messaggio = 'Impossibile caricare l\'elenco degli step per questo esercizio'
) => ({
  type: CREA_VERIFICHE_PREVIEW_ESERCIZIO_FETCH_ERROR,
  messaggio,
});

export const creaverificheVerificaEserciziSet = (payload) => ({
  type: CREA_VERIFICHE_SET,
  payload,
});

export const creaverificheVerificaEserciziSpinnerSet = (enabled) => ({
  type: CREA_VERIFICHE_SPINNER_SET,
  enabled,
});

export const creaverificheVerificaSet = (payload) => ({
  type: CREA_VERIFICHE_VERIFICA_SET,
  payload,
});

export const creaverificheVerificaEsercizioAdd = (step) => ({
  type: CREA_VERIFICHE_VERIFICA_ESERCIZIO_ADD,
  step,
});

export const creaverificheVerificaEsercizioRemove = (step) => ({
  type: CREA_VERIFICHE_VERIFICA_ESERCIZIO_REMOVE,
  step,
});

export const creaverificheVerificaRiepilogoView = (enabled) => ({
  type: CREA_VERIFICHE_VERIFICA_RIEPILOGO_VIEW,
  enabled,
});
