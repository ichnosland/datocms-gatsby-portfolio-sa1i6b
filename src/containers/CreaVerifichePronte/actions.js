/*
 *
 * CreaVerifichePronte actions
 *
 */

import {
  CREA_VERIFICHE_PRONTE_FETCH,
  CREA_VERIFICHE_PRONTE_FETCH_ERROR,
  CREA_VERIFICHE_PRONTE_SET,
  CREA_VERIFICHE_PRONTE_SPINNER_SET,
  CREA_VERIFICHE_PRONTE_FETCH_ESERCIZI,
} from './constants';

export const creaverificheVerifichePronteFetch = (disciplinaId) => ({
  type: CREA_VERIFICHE_PRONTE_FETCH,
  disciplinaId,
});

export const creaverificheVerifichePronteFetchError = (messaggio = 'Impossibile caricare l\'elenco delle verifiche pronte') => ({
  type: CREA_VERIFICHE_PRONTE_FETCH_ERROR,
  messaggio,
});

export const creaverificheVerifichePronteSet = (payload) => ({
  type: CREA_VERIFICHE_PRONTE_SET,
  payload,
});

export const creaverificheVerifichePronteSpinnerSet = (enabled) => ({
  type: CREA_VERIFICHE_PRONTE_SPINNER_SET,
  enabled,
});

export const creaverificheVerifichePronteEserciziFetch = (payload) => ({
  type: CREA_VERIFICHE_PRONTE_FETCH_ESERCIZI,
  payload,
});
