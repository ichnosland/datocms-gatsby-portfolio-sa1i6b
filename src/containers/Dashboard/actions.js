import {
  DASHBOARD_UNITA_FILTRA,
  DASHBOARD_LIVELLI_SET,
  DASHBOARD_LIVELLI_FETCH,
  DASHBOARD_LIVELLI_FETCH_ERROR,
  DASHBOARD_SPINNER_SET,
  DASHBOARD_LIVELLI_RESET,
  DASHBOARD_MENUEGG_SET,
  DASHBOARD_SEARCHACTIVE_SET,
  DASHBOARD_LIVELLI_ISLOADING_SET,
} from './constants';

export function livelliFetch(configuration, isDocente = false, corsoId = undefined) {
  return {
    type: DASHBOARD_LIVELLI_FETCH,
    configuration,
    isDocente,
    corsoId,
  };
}

export function livelliSet(payload) {
  return {
    type: DASHBOARD_LIVELLI_SET,
    payload,
  };
}

export function livelliReset() {
  return {
    type: DASHBOARD_LIVELLI_RESET,
  };
}

export function livelliFetchError(message = 'Errore caricamento') {
  return {
    type: DASHBOARD_LIVELLI_FETCH_ERROR,
    message,
  };
}

export function livelliIsLoadingSet(value) {
  return {
    type: DASHBOARD_LIVELLI_ISLOADING_SET,
    value,
  };
}

export function spinnerSet(payload) {
  return {
    type: DASHBOARD_SPINNER_SET,
    payload,
  };
}

export function unitaFiltra(parolaChiave) {
  return {
    type: DASHBOARD_UNITA_FILTRA,
    filtro: parolaChiave,
  };
}

export const dashboardMenuEggSet = (value) => ({
  type: DASHBOARD_MENUEGG_SET,
  value,
});

export const dashboardSearchActiveSet = (value) => ({
  type: DASHBOARD_SEARCHACTIVE_SET,
  value,
});
