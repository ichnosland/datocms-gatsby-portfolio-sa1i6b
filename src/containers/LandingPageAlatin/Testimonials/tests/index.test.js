import React from 'react';
import { shallow } from 'enzyme';
import { connect } from 'react-redux';
import { compose } from 'redux';

import configureStore from 'configureStore';
import injectReducer from 'utils/injectReducer';
import reducerCurryer from 'common/applications/reducer';
import { StickyWrap } from 'components/StickyFooter';
import ladingPageReducer from 'containers/LandingPage/reducer';
import { landingPageToggleMenu } from 'containers/LandingPage/actions';

import TestimonialsComposed, { Testimonials, Col } from '../index';

const configuration = {
  disciplinaId: 21,
  product: 'landingalatin',
  blocco: false,
  titoloApplicazione: 'Alatin Academy',
  Lyceum: '/',
  hasClassi: false,
};

const mockProps = {
  configuration,
  landingPage: {
    isMenuOpened: true,
  },
  onlandingPageToggleMenu: () => {},
  match: {
    path: '/path',
  },
};

describe('<TestimonialsComposed />', () => {
  it('should render a StickyWrap', () => {
    const renderedComponent = shallow(<Testimonials {...mockProps} />);
    expect(renderedComponent.find(StickyWrap).length).toBe(1);
    expect(renderedComponent).toMatchSnapshot();
  });
});

describe('<TestimonialsComposed />', () => {
  const withReducerConfiguration = injectReducer({ key: 'configuration', reducer: reducerCurryer(configuration) });
  const withLandingConfiguration = injectReducer({ key: 'landingPage', reducer: ladingPageReducer });
  const ConfigurationWrapper = compose(
    withReducerConfiguration,
    withLandingConfiguration,
    connect()
  )(TestimonialsComposed);
  const mockStore = configureStore({}, {});
  mockStore.dispatch = jest.fn();
  it('onModalSetData deve chiamare modalSetData', () => {
    const renderedComponent = shallow(
      <ConfigurationWrapper match={mockProps.match} />, { context: { store: mockStore } }
    ).dive().dive().dive();

    const receivedProps = renderedComponent.props();
    expect(receivedProps.configuration).toEqual(
      configuration
    );

    expect(mockStore.dispatch).not.toHaveBeenCalledWith(
      landingPageToggleMenu(true)
    );
    receivedProps.onlandingPageToggleMenu(true);
    expect(mockStore.dispatch).toHaveBeenCalledWith(
      landingPageToggleMenu(true)
    );
  });
});


describe('<Col />', () => {
  it('should render an <div> tag', () => {
    const renderedComponent = shallow(<Col />);
    expect(renderedComponent.type()).toEqual('div');
  });

  it('should render its css w/ props due', () => {
    const renderedComponent = shallow(<Col due />);
    expect(renderedComponent).toMatchSnapshot();
  });

  it('should render its css w/ props tre', () => {
    const renderedComponent = shallow(<Col tre />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
