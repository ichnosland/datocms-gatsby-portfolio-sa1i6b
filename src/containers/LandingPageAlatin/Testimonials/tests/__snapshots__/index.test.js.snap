// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Col /> should render its css w/ props due 1`] = `
.c0 {
  display: inline-block;
  padding: 0 10px;
  vertical-align: top;
  width: 50%;
}

@media (max-width:41.6875em) {
  .c0 {
    display: block;
    width: 100%;
  }
}

<div
  className="c0"
/>
`;

exports[`<Col /> should render its css w/ props tre 1`] = `
.c0 {
  display: inline-block;
  padding: 0 10px;
  vertical-align: top;
  width: 33.33333%;
}

@media (max-width:41.6875em) {
  .c0 {
    display: block;
    width: 100%;
  }
}

<div
  className="c0"
/>
`;

exports[`<TestimonialsComposed /> should render a StickyWrap 1`] = `
<StickyFooter__StickyWrap>
  <StickyFooter__StickyTop>
    <ComponentWithScrollPosition />
    <Section
      bgColor="#00abe5"
      direction="column"
      first={true}
    >
      <Container
        align="center"
      >
        <Heading__H1
          color="#fff"
          margin="20px 0 0"
        >
          Testimonianze
        </Heading__H1>
      </Container>
      <Container
        align="center"
      >
        <Heading__H4
          color="#000"
          padding="36px 0"
        >
          I video del Convegno:
        </Heading__H4>
        <Testimonials__ColWrap>
          <Testimonials__Col
            tre={true}
          >
            <Testimonials__VideoBox>
              <Testimonials__FlexMedia>
                <iframe
                  allowFullScreen={true}
                  frameBorder="0"
                  height="315"
                  src="https://www.youtube.com/embed/W6Oqv9AaTvo?rel=0"
                  title="Lo stato del latino in Italia: i dati"
                  width="560"
                />
              </Testimonials__FlexMedia>
            </Testimonials__VideoBox>
            <Testimonials__Caption>
              Lo stato del latino in Italia: i dati
            </Testimonials__Caption>
          </Testimonials__Col>
          <Testimonials__Col
            tre={true}
          >
            <Testimonials__VideoBox>
              <Testimonials__FlexMedia>
                <iframe
                  allowFullScreen={true}
                  frameBorder="0"
                  height="315"
                  src="https://www.youtube.com/embed/5inrptQIBSw?rel=0"
                  title="Alatin al liceo Classico"
                  width="560"
                />
              </Testimonials__FlexMedia>
            </Testimonials__VideoBox>
            <Testimonials__Caption>
              Alatin al liceo Classico
            </Testimonials__Caption>
          </Testimonials__Col>
          <Testimonials__Col
            tre={true}
          >
            <Testimonials__VideoBox>
              <Testimonials__FlexMedia>
                <iframe
                  allowFullScreen={true}
                  frameBorder="0"
                  height="315"
                  src="https://www.youtube.com/embed/fO7ZMkpln4U?rel=0"
                  title="Alatin al liceo Scentifico"
                  width="560"
                />
              </Testimonials__FlexMedia>
            </Testimonials__VideoBox>
            <Testimonials__Caption>
              Alatin al liceo Scentifico
            </Testimonials__Caption>
          </Testimonials__Col>
        </Testimonials__ColWrap>
      </Container>
    </Section>
    <Section
      bgColor="#00abe5"
    >
      <Container
        align="center"
      >
        <Testimonials__ColWrap>
          <Testimonials__Col
            tre={true}
          >
            <Testimonials__VideoBox>
              <Testimonials__FlexMedia>
                <iframe
                  allowFullScreen={true}
                  frameBorder="0"
                  height="315"
                  src="https://www.youtube.com/embed/dhHJIW6m8-E?rel=0"
                  title="Alatin per il triennio: Testimonials"
                  width="560"
                />
              </Testimonials__FlexMedia>
            </Testimonials__VideoBox>
            <Testimonials__Caption>
              Alatin per il triennio: Testimonials
            </Testimonials__Caption>
          </Testimonials__Col>
          <Testimonials__Col
            tre={true}
          >
            <Testimonials__VideoBox>
              <Testimonials__FlexMedia>
                <iframe
                  allowFullScreen={true}
                  frameBorder="0"
                  height="315"
                  src="https://www.youtube.com/embed/I8LATKc-8C0?rel=0"
                  title="Intervista TGR"
                  width="560"
                />
              </Testimonials__FlexMedia>
            </Testimonials__VideoBox>
            <Testimonials__Caption>
              Intervista TGR
            </Testimonials__Caption>
          </Testimonials__Col>
        </Testimonials__ColWrap>
      </Container>
    </Section>
    <Section
      bgColor="#00abe5"
    >
      <Container
        align="center"
      >
        <Heading__H4
          color="#000"
          padding="36px 0"
        >
          I materiali:
        </Heading__H4>
        <Testimonials__ColWrap>
          <Testimonials__Col
            due={true}
          >
            <Testimonials__A
              href="handout_convegno.pdf"
              target="_blank"
            >
              <div>
                <RespImg
                  maxWidth="30%"
                  src="IMAGE_MOCK"
                />
              </div>
              <Testimonials__Caption>
                Handout convegno
              </Testimonials__Caption>
            </Testimonials__A>
          </Testimonials__Col>
          <Testimonials__Col
            due={true}
          >
            <Testimonials__A
              href="intervista_redana.pdf"
              target="_blank"
            >
              <div>
                <RespImg
                  maxWidth="30%"
                  src="IMAGE_MOCK"
                />
              </div>
              <Testimonials__Caption>
                Intervista al professore
              </Testimonials__Caption>
            </Testimonials__A>
          </Testimonials__Col>
        </Testimonials__ColWrap>
      </Container>
    </Section>
    <Section
      bgColor="#00abe5"
    >
      <Container
        align="center"
        padding="0 0 60px 0"
      >
        <Heading__H4
          color="#000"
          padding="36px 0"
        >
          Rassegna stampa:
        </Heading__H4>
        <Testimonials__ColWrap>
          <Testimonials__Col
            due={true}
          >
            <Testimonials__A
              href="http://www.lastampa.it/2018/02/26/societa/sfregando-la-lampada-di-alatin-il-latino-diventa-un-gioco-online-z9gSIKOefTv9XW0IpNTyWL/pagina.html"
              target="_blank"
            >
              <div>
                <RespImg
                  maxWidth="90%"
                  src="IMAGE_MOCK"
                />
              </div>
              <Testimonials__Caption>
                La Stampa.it
              </Testimonials__Caption>
            </Testimonials__A>
          </Testimonials__Col>
          <Testimonials__Col
            due={true}
          >
            <Testimonials__A
              href="http://torino.repubblica.it/cronaca/2018/02/27/news/ora_i_latino_a_scuola_si_studia_con_lo_smartphone_e_copiare_i_compiti_non_conviene-189916501"
              target="_blank"
            >
              <div>
                <RespImg
                  maxWidth="90%"
                  src="IMAGE_MOCK"
                />
              </div>
              <Testimonials__Caption>
                Repubblica.it
              </Testimonials__Caption>
            </Testimonials__A>
          </Testimonials__Col>
        </Testimonials__ColWrap>
      </Container>
    </Section>
  </StickyFooter__StickyTop>
  <StickyFooter__StickyBottom>
    <ZigZagBorder />
    <Footer />
  </StickyFooter__StickyBottom>
</StickyFooter__StickyWrap>
`;
